dnsaas/urls.py

```diff
@@ -8,11 +7,19 @@
 from ui.views import start_page
 
 
+def patterns(prefix, *args):
+    if VERSION < (1, 9):
+        from django.conf.urls import patterns as django_patterns
```

Comment:
If we only care about 1.8+, then yes. I didn't want to make that assumption. I'll keep this in because it isn't doing any harm and I'd prefer to allow all supported Django versions (down to 1.7).